src/content/general/management-clusters/index.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: Management Clusters
+title: Giant Swarm Management Clusters
```

Comment:
```suggestion
title: Management Clusters
```

"Giant Swarm" is not needed in the title (because the context makes it self-evident that we talk about Giant Swarm stuff), but looks weird once it occurs in more and more titles/headlines.

Reply:
Also changed to sentence case